### ISSUE TITLE
Uncaught TypeError: Cannot read property 'id' of undefined

### ISSUE THREAD

**Issue by Kkthnx** [steelbrain/linter-ui-default#357]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.0-beta0 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [linter-ui-default](https://github.com/steelbrain/linter-ui-default) package 1.6.1

### Stack Trace

Uncaught TypeError: Cannot read property 'id' of undefined

```
At C:\Users\kkthn\AppData\Local\atom\app-1.19.0-beta0\resources\app\src\text-editor-component.js:2132

TypeError: Cannot read property 'id' of undefined
    at horizontalPositionsToMeasure.forEach (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/text-editor-component.js:2132:73)
    at Map.forEach (native)
    at TextEditorComponent.measureHorizontalPositions (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/text-editor-component.js:2128:45)
    at TextEditorComponent.measureContentDuringUpdateSync (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/text-editor-component.js:373:16)
    at TextEditorComponent.updateSync (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/text-editor-component.js:270:18)
    at TextEditorComponent.pixelPositionForScreenPosition (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/text-editor-component.js:194:18)
    at HTMLElement.pixelPositionForScreenPosition (~/AppData/Local/atom/app-1.19.0-beta0/resources/app/src/text-editor-element.js:243:32)
    at getBufferPositionFromMouseEvent (/packages/linter-ui-default/lib/editor/helpers.js:12:47)
    at /packages/linter-ui-default/lib/editor/index.js:163:29)
    at later (/packages/linter/node_modules/sb-debounce/index.js:9:14)
```

### Commands

```
     -0:23.2.0 core:select-all (input.hidden-input)
     -0:22.9.0 core:copy (input.hidden-input)
     -0:20 intentions:highlight (input.hidden-input)
     -0:19.9.0 core:select-all (input.hidden-input)
     -0:19.6.0 core:copy (input.hidden-input)
     -0:15.1.0 core:paste (div.line)
     -0:14.6.0 intentions:highlight (input.hidden-input)
     -0:14.5.0 core:select-all (input.hidden-input)
     -0:14.2.0 core:save (input.hidden-input)
     -0:12 intentions:highlight (input.hidden-input)
     -0:11.9.0 core:select-all (input.hidden-input)
     -0:11.6.0 core:copy (input.hidden-input)
     -0:07.4.0 core:paste (div.line)
     -0:06.4.0 intentions:highlight (input.hidden-input)
     -0:06.2.0 core:select-all (input.hidden-input)
     -0:06.0 core:save (input.hidden-input)
```

### Non-Core Packages

```
busy-signal 1.4.3 
intentions 1.1.2 
language-lua-wow 0.2.0 
language-toc-wow 0.2.0 
linter 2.1.4 
linter-lua-findglobals 1.2.0 
linter-ui-default 1.6.1 
sort-lines 0.14.0 
```

**Comment by steelbrain** [comment]
Also on Atom beta

**Comment by Arcanemagus** [comment]
Finally had this happen to me, can confirm it's an internal Atom issue as all the input to the API functions is valid as far as `linter-ui-default` can do.

**Comment by Arcanemagus** [comment]
For reference this particular issue should have been fixed by https://github.com/atom/atom/pull/15051, released in Atom v1.20.0.
